.github/workflows/github-page.yml

```diff
@@ -70,6 +65,11 @@ jobs:
     - name: Cabal Configure
       run: cabal configure --builddir="$CABAL_BUILDDIR" --write-ghc-environment-files=always
 
+    - name: Use cabal.project.local.github-pages
```

Comment:
Why this move?

Reply:
Ah, yes.  This is correct.  Unfortunately `cabal configure` clobbers `cabal.project.local` so this has to go after `cabal.project.local`.

Reply:
Yes. One can see the effect in https://github.com/input-output-hk/cardano-node/runs/2173237313